notes/20170814_meeting.md

```diff
@@ -0,0 +1,48 @@
+# GraphQL WG Meeting #1 Notes (August 14, 2017)
```

Comment:
Could we rename this file to `2017-08-14.md` to match the naming scheme in https://github.com/graphql/graphql-wg/tree/master/agendas